recipes/libglvnd/libglvnd_look_for_egl_icd_in_system_location.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/src/EGL/meson.build b/src/EGL/meson.build
+index c90f0ca..3cf618a 100644
+--- a/src/EGL/meson.build
++++ b/src/EGL/meson.build
+@@ -39,9 +39,12 @@ if host_machine.system() in ['haiku']
+   egl_vendor_config_dirs += ':/boot/system/non-packaged/add-ons/opengl/egl_vendor.d'
+   egl_vendor_config_dirs += ':/boot/system/add-ons/opengl/egl_vendor.d'
+ else
++  # conda-forge specific patch: in conda-forge we do not ship any EGL implementation
++  # as a conda-forge package, so we patch the DEFAULT_EGL_VENDOR_CONFIG_DIRS to point
++  # to the system one, where we assume that glvnd is installed with /usr prefix
+   egl_vendor_config_dirs = '@0@/glvnd/egl_vendor.d:@1@/glvnd/egl_vendor.d'.format(
+-    join_paths(get_option('prefix'),get_option('sysconfdir')),
+-    join_paths(get_option('prefix'),get_option('datadir')))
++    join_paths('/usr',get_option('sysconfdir')),
```

Comment:
This gets resolved at build time to be either lib or lib64 correct? Wont this cause problems between Ubuntu and fedora

Reply:
Fortunately not, this are not the directory that contains the libraries, but rather the directories that contains the `.json` configuration files containg ICD information, see https://github.com/NVIDIA/libglvnd/blob/master/src/EGL/icd_enumeration.md . 
So by default they should be `/usr/share/glvnd/egl_vendor.d` and `/etc/glvnd/egl_vendor.d/`. To be honest, I need to actually check if `/etc/glvnd/egl_vendor.d/` is working as intended, as it is not clear how `/etc/glvnd/egl_vendor.d/` could emerge if `prefix` is `/usr`, but `/usr/share/glvnd/egl_vendor.d` is working fine and should be the same across Ubuntu and Fedora.

Reply:
https://gitlab.freedesktop.org/glvnd/libglvnd/-/blob/v1.7.0/src/EGL/Makefile.am#L53

`/etc/glvnd/egl_vendor.d/` emerges because of here.